configs/mock/rollup.toml

```diff
@@ -71,4 +72,12 @@ disable_state_root_consistency_checks = true
 # "None" - Shutdown the sequencer instead of attempting recovery (default)
 # "TryToSave" - Attempt to recover by flushing batches and catching up with the chain
 recovery_strategy = "TryToSave"
-batch_execution_time_limit_millis = 6000 # This should be adjusted depending on the DA layer's block times
+batch_execution_time_limit_millis = 3000 # This should be adjusted depending on the DA layer's block times
```

Comment:
da block production is every 6 seconds and this is 3 seconds. isnt that too conservative?

Reply:
at least in celestia config these are 1:1

Reply:
Yeah, that got screwed up in the merge with nightly (Theodore had adjusted the block time). Fixed.